ArcadiaParties.API/Properties/launchSettings.json

```diff
@@ -21,10 +21,10 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "weatherforecast",
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "applicationUrl": "http://localhost:5000",
```

Comment:
Removed URL will be used if you enable https in application startup. It is not needed to remove this.

Reply:
Keep as is